src/config.json

```diff
@@ -2,10 +2,12 @@
     "command_prefix": "!",
     "command_resolve": "resolve",
     "command_close": "close",
+    "command_end": "end",
     "command_escalate": "escalate",
     "command_bug": "bug",
     "command_sc_resolve": "res",
     "command_sc_close": "cls",
+    "command_sc_close_no_mention": "!cls",
```

Comment:
Is this still visible from the code base? I don't see this. @kien-ngo

Reply:
It was mistakenly added and I removed it. You can view all the changes here: https://github.com/thirdweb-dev/support-discord-bot/pull/68/files